### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by Frityet** [OneMoreGres/ScreenTranslator#153]
Not an issue, just discovered this program today and saw the depreciation message. Thank you for making this awesome program

**Comment by OneMoreGres** [comment]
Thank you too! Words like this really help to keep motivation!

**Comment by sariouso** [comment]
Agreed, thank you so much for this program, makes reading manga so much easier.

**Comment by DaLimCodes** [comment]
Came here to also say thank you so much for this app, @OneMoreGres !

Out of curiosity, how open are you with people making a fork of this and continuing this on their own repo (or making pull request)?

**Comment by OneMoreGres** [comment]
Thank you!

@sariouso

I am not stopping the maintenance completely. I leave it the same as it was last year (minor fixes and delays). Just clarified it in the readme

@DaLimCodes 

It would be great if someone forked and developed it. I can add links to forks in the readme. Pull requests usually take a time to process, so a fork would be better

**Comment by joaohw601** [comment]
I also really liked the project, but if somehow it translates simultaneously this program would definitely be the most used both in movies and games. I really hope it continues
